doc/api_entity.md

```diff
@@ -216,17 +216,17 @@ A boolean indicating if validation is enabled or not.
 
 ---------
 
-## .getValidation()
+## .getController() (useless)
```

Comment:
i'll up some updates, but think about this again. the main problem sortof comes with the way we register entities in the configure function, since we either need to make that the aurelia-validation is initialized (but not with use.plugin('aurelia-validation') since that's far behind the 'import * from 'entities' at the top), or use the loader like i did in some aurelia-config version to load then entities ourself instead of using import. that would probably the better version